Source/drlg_l3.cpp

```diff
@@ -1307,6 +1310,7 @@ void DRLG_L3PlaceRndSet(const BYTE *miniset, int rndper)
 			kk = sw * sh + 2;
 			if (miniset[kk] >= 84 && miniset[kk] <= 100 && found == TRUE) {
 				// BUGFIX: accesses to dungeon can go out of bounds
+				// BUGFIX: Comparisons vs 100 should use same tile as comparisons vs 84.

```

Comment:
A clear copy paste error by the original developer.

The range 84-100 appears to contain alternative variations of some wall pieces. This code is obviously intended to stop the variant pieces from being placed next to each other. Similar code exists in the other PlaceMiniSet functions.